### PR TITLE
4.20.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Review by blackbaud-johnly — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 4.20.0 (2021-07-01)
+
+- Added a layout to the input box component that supports the search component in modern theme. [#273](https://github.com/blackbaud/skyux-forms/pull/273)
+- Fixed the input box component to have proper spacing with the country field component. [#273](https://github.com/blackbaud/skyux-forms/pull/273)
```
Comment (blackbaud-paulah) [inline CHANGELOG.md:4]:
```suggestion
- Fixed the input box component to have proper spacing within the country field component. [#273](https://github.com/blackbaud/skyux-forms/pull/273)
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-4.20.0-release](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-4.20.0-release)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-4.20.0-release](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-4.20.0-release)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-4.20.0-release](https://developer.blackbaud.com/skyux-forms?_sp.spa=pr-4.20.0-release)

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**